easybuild/easyconfigs/r/rpy2/rpy2-3.4.5-foss-2020b.eb

```diff
@@ -0,0 +1,60 @@
+# Author: Pavel Grochal (INUITS)
+# Updated: Denis Kristak (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'rpy2'
+version = '3.4.5'
+
+homepage = 'https://rpy2.bitbucket.io/'
+description = """rpy2 is an interface to R running embedded in a Python process."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('R', '4.0.3'),
+    ('IPython', '7.18.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('simplegeneric', '0.8.1', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['dc972e06094b9af5b855b3df4a646395e43d1c9d0d39ed345b7393560d0b9173'],
+    }),
+    ('tzlocal', '4.1', {
+        'checksums': ['0f28015ac68a5c067210400a9197fc5d36ba9bc3f8eaf1da3cbd59acdfed9e09'],
+    }),
+    ('cffi', '1.15.0', {
+        'checksums': ['920f0d66a896c2d99f0adbb391f990a84091179542c205fa53ce5787aff87954'],
+    }),
+    ('tzdata', '2021.5', {
+        'checksums': ['68dbe41afd01b867894bbdfd54fa03f468cfa4f0086bfb4adcd8de8f24f3ee21'],
+    }),
+    ('backports-zoneinfo', '0.2.1', {
+        'modulename': 'backports.zoneinfo',
+        'source_tmpl': 'backports.zoneinfo-%(version)s.tar.gz',
+        'checksums': ['fadbfe37f74051d024037f223b8e001611eac868b5c5b06144ef4d8b799862f2'],
+    }),
+    ('pytz-deprecation-shim', '0.1.0.post0', {
+        'source_tmpl': 'pytz_deprecation_shim-%(version)s.tar.gz',
+        'checksums': ['af097bae1b616dde5c5744441e2ddc69e74dfdcb0c263129610d85b87445a59d'],
+    }),
+    (name, version, {
+        'checksums': ['5d31a5ea43f5a59f6dec30faca87edb01fc9b8affa0beae96a99be923bd7dab3'],
+    }),
+]
+
+# sanity_check_commands = [
+#     "pytest --pyargs rpy2 "
```

Comment:
Why are these commented out?

Reply:
@boegel it is because even though i disable those tests, they still run (and fail...) ... should figure out how to disable them for real....

Reply:
If I uncomment that, I'm still getting
`FAILED tests/ipython/test_rmagic.py` so the `--ignore` thingy is not working...i'll try to figure it out

Reply:
i had to figure out how to do this properly, because the above didnt prevent tests from running (and failing)...its updated now